playbooks/os_updates.yml

```diff
@@ -87,3 +94,15 @@
         token: "{{ vault_pul_slack_token }}"
         msg: "Ansible ran `{{ ansible_play_name }}` on {{ inventory_hostname }}"
         channel: "{{ slack_alerts_channel }}"
+
+  post_tasks:
+    - name: notify ops to reboot the ansible servers if needed
```

Comment:
we will soon be getting Rocky VMs and this will get them also.

Reply:
as long as the Rocky VMs return `RedHat` as their `os_family` they should be updated by the `clear local repository` task and rebooted by the `Reboot if required` task

Reply:
Confirmed today that the Rocky VMs do report their `os_family` as `RedHat`, so they are updated correctly.